src/main/java/eu/bbmri/eric/csit/service/negotiator/database/model/Post.java

```diff
@@ -46,5 +46,8 @@ public class Post extends AuditEntity {
   private Resource resource;
 
   @Lob private String text;
-  private String status;
+
+  private PostStatus status;
```

Comment:
PLease change the persistence of Enums to enum not int. In postgres it is saved as an int.